src/script/event/EventRepository.ts

```diff
@@ -184,18 +186,15 @@ export class EventRepository {
       this.disconnectWebSocket?.();
       return new Promise<void>(async resolve => {
         this.disconnectWebSocket = await account.listen({
-          onConnected: () => {
-            this.notificationHandlingState(NOTIFICATION_HANDLING_STATE.WEB_SOCKET);
-            this.updateConnectivitityStatus(WEBSOCKET_STATE.OPEN);
-            resolve();
+          onConnectionStateChanged: connectionState => {
+            this.updateConnectivitityStatus(connectionState);
+            if (connectionState === ConnectionState.LIVE) {
+              resolve();
+            }
           },
-          onConnectionStateChanged: this.updateConnectivitityStatus,
           onEvent: this.handleIncomingEvent,
           onMissedNotifications: this.triggerMissedSystemEventMessageRendering,
-          onNotificationStreamProgress: ({done, total}) => {
-            amplify.publish(WebAppEvents.WARNING.SHOW, Warnings.TYPE.CONNECTIVITY_RECOVERY);
```

Comment:
This was a mistake. It triggered the banner everytime we process one message from the notification stream. 
Not a big deal, since it's reduced to 1 event, but semantically incorrect